src/HotChocolate/Data/test/Data.Projections.Tests/IntegrationTests.cs

```diff
@@ -289,6 +339,24 @@ public IQueryable<Foo> Foos
         => new Foo[] { new() { Bar = "A" }, new() { Bar = "B" } }.AsQueryable();
 }
 
+public class Mutation
+{
+    [UseMutationConvention]
+    [UseProjection]
+    public IQueryable<Foo> Modify()
```

Comment:
Can we have one test here that shows that it also works when there is an error? I mean, that it not just blows up completely

Reply:
Done